src/requests/getForecast.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+import axios from "axios";
+
+const getForecast = (
+  searchText,
+  setSelectedDate,
+  setForecasts,
+  setLocation,
+  setErrorMessage
+) => {
+  let endpoint = "https://cmd-shift-weather-app.onrender.com/forecast";
+
+  if (searchText) {
+    endpoint += `?city=${searchText}`;
+  }
+
+  axios
+    .get(endpoint)
+    .then((response) => {
+      setSelectedDate(response.data.forecasts[0].date);
+      setForecasts(response.data.forecasts);
+      setLocation(response.data.location);
```

Comment:
You are missing setErrorMessage("") from here. 
This is important because if you type in Liverpooool accidentally that will return an error 404, and on line 26 you say if (status === 404) set the error message to "No such town or city".
Then you will delete that and type in the correct city Liverpool, you get back a response and set all the correct data (on line 20-22), but you still have an error message which you haven't cleared therefore in App.js (line 56-63) you will display the error message instead of the ForecastDetails.